examples/packer-basic-example/README.md

```diff
@@ -37,4 +37,41 @@ Terratest. For slightly more complicated, real-world examples of Packer template
 1. Install [Golang](https://golang.org/) and make sure this code is checked out into your `GOPATH`.
 1. `cd test`
 1. `dep ensure`
-1. `go test -v -run TestPackerBasicExample`
\ No newline at end of file
+1. `go test -v -run TestPackerBasicExample`
+
+
+
+
+## Running automated tests against this Packer template for the GCP builder
+
+1. Sign up for [GCP](https://cloud.google.com/).
+1. Configure your GCP credentials using one of the
+   [Authentication](https://www.packer.io/docs/builders/googlecompute.html#authentication) methods.
+1. Install [Packer](https://www.packer.io/) and make sure it's on your `PATH`.
+1. Install [Golang](https://golang.org/) and make sure this code is checked out into your `GOPATH`.
+1. `cd test`
+1. `dep ensure`
+1. `go test -v -run TestPackerGCPBasicExample`
+
+
+
+
+## Running automated tests against this Packer template for the OCI builder
+
+1. Sign up for [OCI](https://cloud.oracle.com/cloud-infrastructure).
+1. Configure your OCI credentials via [CLI Configuration
+   Information](https://docs.cloud.oracle.com/iaas/Content/API/Concepts/sdkconfig.htm).
+1. Create following [Packer-required](https://www.packer.io/docs/builders/oracle-oci.html#required)
+   resources in your tenancy:
+   1. [Compartment](https://docs.cloud.oracle.com/iaas/Content/GSG/Tasks/choosingcompartments.htm)
+   1. [VCN](https://docs.cloud.oracle.com/iaas/Content/GSG/Tasks/creatingnetwork.htm) and subnet
+1. Create following environment properties:
+   1. `TF_VAR_compartment_ocid` with the OCID of the previously created compartment.
+   1. `TF_VAR_pass_phrase` with the pass phrase for decrypting of the OCI [API signing
+      key](https://docs.cloud.oracle.com/iaas/Content/API/Concepts/apisigningkey.htm) (can be omitted
+      if the key is not protected).
```

Comment:
Seems like a lot of steps! Does OCI not provide defaults for compartment or VCN we could leverage, like the default VPC in AWS?

Reply:
I checked OCI documentation:

* We can use _tenancy_ as a (root) compartment - I'll modify test for this.
* Unfortunately, there is no default VCN and it has to be created. So, it's a mandatory step. (Of course, this can be later simplified with _Terraform_ which I plan as the next test.)

Reply:
Understood, thx!